### PR TITLE
Bulk upgrade core dependencies

### PR DESCRIPTION
**Pull request by jason-codaio**
#### Is this adding or improving a _feature_ or fixing a _bug_?

Upgrading dependencies. Required minor test update.

#### What's the new behavior?

Hopefully none.

#### Have you checked that...?

- [X] The new code matches the existing patterns and styles.
- [X] The tests pass with `yarn test`.
- [X] The linter passes with `yarn lint`. (Fix errors with `yarn fix`.)
- [X] The relevant examples still work. (Run examples with `yarn start`.)

#### Does this fix any issues or need any specific reviewers?

Reviewers: @timbuckley

### CODE REVIEW

File: site/pages/examples/[example].tsx [hunk 1]
```diff
@@ -256,6 +267,7 @@ const ExamplePage = ({ example }: { example: string }) => {
   const [name, Component, path] = EXAMPLE
   return (
     <ErrorBoundary
+      FallbackComponent={ErrorFallback}
```
Comment (timbuckley) [inline site/pages/examples/[example].tsx:270]: Did you get a chance to manually test the result of an error here?

Comment (jason-krypton) [reply site/pages/examples/[example].tsx:270]: Honestly no, but I just copy pasted this off the package Readme

Comment (jason-krypton) [reply site/pages/examples/[example].tsx:270]: My assumption was this is not hit and I'd have to break one of the examples to hit this. Also given its test/examples code I was less concerned

**Comment by jason-krypton** [comment]
Is this good to go? @timbuckley

**Pull request closed without merging**